### ISSUE TITLE
Observed overparameterized in CIFAR10

### ISSUE THREAD

**Issue by oovm** [GalAster/WLNet-Model-Zoo#4]
# WRN28-10 trained on CIFAR10
![Task](https://img.shields.io/badge/Task-Classifation-Orange.svg)
![Size](https://img.shields.io/badge/Size-145.98%20MB-blue.svg)
![Accuracy](https://img.shields.io/badge/Accuracy-100.00%25-brightgreen.svg)
![Speed](https://img.shields.io/badge/Speed-3.603%20ms-ff69b4.svg)

Automatically generated on Mon 19 Nov 2018 13:05:14

## Network structure:
- Network Size: **145.989 MB**
- Parameters: **36 497 222**
- Nodes Count: **95**
- Speed: **3.603 ms/sample**
- Layers:
  - AggregationLayer: **1**
  - BatchNormalizationLayer: **27**
  - ConvolutionLayer: **28**
  - ElementwiseLayer: **25**
  - LinearLayer: **1**
  - SoftmaxLayer: **1**
  - ThreadingLayer: **12**

## Accuracy Curve
![classification curve](https://user-images.githubusercontent.com/17541209/48687344-d6ed6180-ebfb-11e8-8358-12a889603c4a.png)

![high precision classification curve](https://user-images.githubusercontent.com/17541209/48687347-d785f800-ebfb-11e8-8082-be738d544b14.png)

## Main Indicator
  - Top-1: **100.000%**
  - Top-2: **100.000%**
  - Top-3: **100.000%**
  - Top-5: **100.000%**
  - LogLikelihood: **-65.802**
  - CrossEntropyLoss: **0.00131604**
  - ProbabilityLoss: **0.00131353**
  - MeanProbability: **99.8686%**
  - GeometricMeanProbability: **99.8684%**
  - VarianceProbability: **0.00000296**
  - ScottPi: **1.**
  - CohenKappa: **1.**
  - RejectionRate: **0.00000%**

![accuracy rejection curve](https://user-images.githubusercontent.com/17541209/48687348-d785f800-ebfb-11e8-9304-feb6311c4869.png)

## Class Indicator
| Class | Count | TPRate | TNRate | FPRate | FNRate | F1Score |
|-------|-------|--------|--------|--------|--------|---------|
| airplane | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| automobile | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| bird | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| cat | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| deer | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| dog | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| frog | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| horse | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| ship | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |
| truck | 5000 | 100.000% | 100.000% | 0.00000% | 0.00000% | 1.00000 |

## Hard Class
![confusionmatrix](https://user-images.githubusercontent.com/17541209/48687346-d785f800-ebfb-11e8-9c34-66a28c13f5ff.png)

## Evaluation Report
| Index | TestID | Result | Time | MemoryChange |
|-------|--------|--------|------|--------------|
| 1 | Dependency Check | Success | 2.39414 s | +0.03443 MB |
| 2 | Pre-define | Success | 0.01592 s | -152.407 MB |
| 3 | GPU Warm-Up | Success | 2.03158 s | +0.03586 MB |
| 4 | Loading Model | Success | 1.62876 s | +146.398 MB |
| 5 | Loading Data | Success | 2.67280 s | +175.884 MB |
| 6 | Benchmark Test | Success | 181.989 s | +10.9483 MB |
| 7 | Result Dump | Success | 0.17468 s | -0.08306 MB |
| 8 | Analyzing | Success | 23.6474 s | +0.26517 MB |
